x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/use_preview_histogram.tsx

```diff
@@ -37,7 +37,7 @@ export const usePreviewHistogram = ({
     config: getEsQueryConfig(uiSettings),
     indexPattern: {
       fields: [],
-      title: index.join(),
+      title: index == null ? [].join() : index.join(),
```

Comment:
Maybe it will be more clear?
```suggestion
      title: index == null ? '' : index.join(),
```